src/models/account.cpp

```diff
@@ -36,8 +36,13 @@ unsigned int stoui(const std::string& str, size_t* idx = nullptr, int base = 10)
 Account::Account(const std::string& path) : m_path{ path }, m_db{ std::make_shared<SQLite::Database>(m_path, SQLite::OPEN_READWRITE | SQLite::OPEN_CREATE) }
 {
     //Load Groups
-    m_db->exec("CREATE TABLE IF NOT EXISTS groups (id INTEGER PRIMARY KEY, name TEXT, description TEXT, balance TEXT)");
-    SQLite::Statement qryGetAllGroups{ *m_db, "SELECT * FROM groups" };
+    m_db->exec("CREATE TABLE IF NOT EXISTS groups (id INTEGER PRIMARY KEY, name TEXT, description TEXT)");
+    try
+    {
+        m_db->exec("ALTER TABLE groups DROP COLUMN balance");
```

Comment:
I don't think this 
```cpp
try
{
     m_db->exec("ALTER TABLE groups DROP COLUMN balance");
}
catch(...) {  }
```
is needed, since there is no `Groups` table in the V2022.10.0 stable and I'm ok with breaking API between betas.